optuna/visualization/_pareto_front.py

```diff
@@ -73,6 +73,10 @@ def objective(trial):
             fig = optuna.visualization.plot_pareto_front(study)
             fig.show()
 
+        .. seealso::
+            To customize the generated diagram afterwards (change title, ...), use the
+            function 'update_layout()' of plotly. See https://plotly.com/python/reference/layout/.
```

Comment:
```suggestion
            function ``update_layout()`` of plotly. See https://plotly.com/python/reference/layout/.
```
@gituser789 How about using a code block? It might look better than simply quoting it.

Reply:
<img width="1029" alt="image" src="https://github.com/optuna/optuna/assets/24619922/460ff39a-2697-45a8-afa3-6540d69e4533">
Thank you, now it looks better.